src/config/index.ts

```diff
@@ -1,21 +1,31 @@
 import { PublicKey } from '@solana/web3.js';
-import { Network } from '../models';
-import * as devnet from './devnet';
-import * as mainnet from './mainnet';
-import * as staging from './staging';
+import { ClientEnv, Network } from '../models';
+import * as devnetProd from './devnet/production';
+import * as mainnetProd from './mainnet/production';
+import * as devnetStaging from './devnet/staging';
 import { Vault, VaultProtoConfig, Token } from './types';
 
 export interface Config {
-  defaultProgramId: PublicKey;
+  programId: PublicKey;
   tokens: Record<string, Token>;
   vaultProtoConfigs: Record<string, VaultProtoConfig>;
   vaults: Record<string, Vault>;
 }
 
-export const Configs: Record<Network, Config> = {
-  [Network.MainnetProd]: mainnet,
-  [Network.DevnetProd]: devnet,
-  [Network.DevnetStaging]: staging,
-};
-
+export function getConfig(network: Network, clientEnv: ClientEnv): Config {
+  switch (network) {
+    case Network.Mainnet:
+      return mainnetProd;
+    case Network.Devnet:
+      switch (clientEnv) {
+        case ClientEnv.Production:
+          return devnetProd;
+        case ClientEnv.Staging:
+          return devnetStaging;
+      }
+    case Network.Localnet:
```

Comment:
Do we not support localnet? Maybe we should expose env vars or something. Can add a low priority ticket and todo here for later

Reply:
Doesn't that make it weird for non-react integrations? 

In react, the env vars have to be pre-pended with "REACT_APP" for them to available, but we don't want to enforce that kind of naming on the SDK since it's not exclusive to a react environment. 

I'll add a ticket so we can look into it.

Reply:
Done